### PR TITLE
Make the readme a bit clearer

### PR DESCRIPTION
**Pull request by ldunnplaymint**
We want to make it so the tutorial readme flows a bit better, it isn't currently explicit what to do on the page.

### CODE REVIEW

File: tutorial/README.md [hunk 1]
```diff
@@ -2,16 +2,27 @@
 
 # A guide to building your own game with Downstream
 
+## Welcome to the learning zone!
+
+This page is your one stop shop for learning how to build on top of Downstream. We recommend that you follow the Tutorials to get up and running. The tutorials will explain how to set up your work space and then run you through small example Islands that focus on a small set of mechanics. 
+
+- Once you feel you are up to speed on what is currently possible on Downstream, you can check out our Existing Maps and Games!
+
+- The Existing Maps are various things that Playmind has created in Downstream. There are some smaller examples like the Gate Demo and some larger examples like the quest-map.
```
Comment (Steedie) [inline tutorial/README.md:11]:
```suggestion
The Existing Maps are various things that Playmint has created in Downstream. There are some smaller examples like the Gate Demo and some larger examples like the quest-map.
```
Playmind -> Playmint

**Review by Steedie — CHANGES_REQUESTED** [review]

**Review by Steedie — APPROVED** [review]

File: tutorial/README.md [hunk 2]
```diff
@@ -2,16 +2,27 @@
 
 # A guide to building your own game with Downstream
 
+## Welcome to the learning zone!
+
+This page is your one stop shop for learning how to build on top of Downstream. We recommend that you follow the Tutorials to get up and running. The tutorials will explain how to set up your work space and then run you through small example Islands that focus on a small set of mechanics. 
+
+- Once you feel you are up to speed on what is currently possible on Downstream, you can check out our Existing Maps and Games!
```
Comment (chrisfarms) [inline tutorial/README.md:9]: a list with only one thing probably doesn't deserve to be a list :)

```suggestion
Once you feel you are up to speed on what is currently possible on Downstream, you can check out our Existing Maps and Games!
```

File: tutorial/README.md [hunk 3]
```diff
@@ -2,16 +2,27 @@
...
+
+The Existing Maps are various things that Playmint has created in Downstream. There are some smaller examples like the Gate Demo and some larger examples like the quest-map.
+The Existing Games are full games that have been developed by external developers and one by Playmint! They will give you an insight into how far Downstream has already been pushed!
```
Comment (chrisfarms) [inline tutorial/README.md:12]: look like you intended this to be a linebreak, but didn't give it the extra space? 

```suggestion

The Existing Games are full games that have been developed by external developers and one by Playmint! They will give you an insight into how far Downstream has already been pushed!
```

File: tutorial/README.md [hunk 4]
```diff
@@ -2,16 +2,27 @@
...
+
+For more in depth information, check out the References page!
```
Comment (chrisfarms) [inline tutorial/README.md:14]: Not sure it's really "more in depth" just different

and should it be a link?

```suggestion
See the [References](./REFERENCE.md) for information on common tooling and commands.
```

or something like that

**Review by chrisfarms — COMMENTED** [review]
personally I would skip over a block of text like this, so there's maybe an argument for making it a little more call-to-actiony rather than introy ... but I think I might be the minority here so probably ignore me.

I've noticed that the references link at the bottom of the page is broken, so prob worth fixing up in this PR ... or does the note in the intro replace the section in the footer?

**Review by chrisfarms — APPROVED** [review]

**Comment by chasensa** [comment]
@ldunnplaymint is it worth adding the Discord support chat link at the end of the intro? 

something like: (feel free to change!) 
Specific questions? Drop into our Discord support chat where you can have direct access to the Playmint dev team: https://discord.gg/MvYb9PBjGW

**Pull request merged**